include/odp/api/spec/packet_io.h

```diff
@@ -845,6 +848,12 @@ typedef struct odp_pktio_capability_t {
 		uint32_t max_output;
 	} maxlen;
 
+	/**
+	 * Max Tx aging timeout in nano seconds supported when packet aging
+	 * feature is supported.
+	 */
+	uint64_t max_tx_aging_tmo_ns;
```

Comment:
This misses to define when aging is / is not supported. E.g.:
0: aging is not supported
>0: maximum aging timeout supported in nanoseconds

Reply:
Ack.